drivers/bridge/bridge.go

```diff
@@ -432,6 +432,45 @@ func parseNetworkOptions(option options.Generic) (*networkConfiguration, error)
 	return config, nil
 }
 
+// Returns the non link-local IPv6 subnet for the containers attached to this bridge if found, nil otherwise
+func getV6Network(config *networkConfiguration, i *bridgeInterface) *net.IPNet {
+	if config.FixedCIDRv6 != nil {
+		return config.FixedCIDRv6
+	}
+
+	if i.bridgeIPv6 != nil && i.bridgeIPv6.IP != nil && !i.bridgeIPv6.IP.IsLinkLocalUnicast() {
+		return i.bridgeIPv6
+	}
+
+	return nil
+}
+
+func isolateNetworks(d *driver, thisV4, thisV6 *net.IPNet, enable bool) error {
+	d.Lock()
+	defer d.Unlock()
```

Comment:
Thanks, that seems it would work.
I need to think a little bit more on what would happen if, after collecting the subnets and in the middle of installing the network isolation rules, somebody removes a network.
I want to make sure we don't end up with stale rules.